pages.id/common/gh.md

```diff
@@ -0,0 +1,37 @@
+# gh
+
+> Memudahkan pengaksesan GitHub dari command-line.
+> Beberapa subcommands seperti `gh config` memiliki dokumentasi sendiri.
+> Informasi lebih lanjut: <https://cli.github.com/>.
+
+- Clone sebuah GitHub repository pada local:
+
+`gh repo clone {{owner}}/{{repository}}`
```

Comment:
Please translate the arguments for get this easier for non-English speakers. Perhaps is better if you translate `repository`.

Reply:
Hi, I leave this not translated in purpose, because usually we call it as is. The literal translation could be unfamiliar. wdyt?

Reply:
How about this?

```
Mengklon sebuah GitHub repositori di lokal
```

Reply:
So, `owner` shouldn't be translated?

Reply:
ah right, missed it.

one sec